src/pages/ChannelPage/index.js

```diff
@@ -0,0 +1,99 @@
+import axios from '../../api/axios';
+import requests from '../../api/requests';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import ChannelRow from '../../components/MainComponents/ChannelRow';
+import styled from 'styled-components';
+const ChannelPage = () => {
+  let { channelId } = useParams();
+  console.log(channelId);
+  const [channelVideo, setChannelVideo] = useState([]);
+  const [channelData, setChannelData] = useState([]);
+  async function getChannelData() {
+    try {
+      const response = await axios(requests.fetchChannelData(channelId));
+      const cdata = response.data.items;
+      setChannelData(cdata);
+      const response2 = await axios(requests.fetchPlayLists(channelId));
+      const vdata = response2.data.items;
+      setChannelVideo(vdata);
+    } catch (error) {
+      console.log(error);
+    }
+  }
```

Comment:
response, response2와 같이 무의미한 변수명을 부여하는것은 좋지 않습니다.
```suggestion
  async function getChannelData() {
    try {
      const {data:  channelData} = await axios(requests.fetchChannelData(channelId));
      setChannelData(channelData.items);
      const {data:  playLists}  = await axios(requests.fetchPlayLists(channelId));
      setChannelVideo(playLists.items);
    } catch (error) {
      console.log(error);
    }
  }
```